src/main/java/com/epam/test_generator/controllers/test_result/TestResultController.java

```diff
@@ -34,17 +36,17 @@
 
     @ApiOperation(value = "Run selected tests", nickname = "runTests")
     @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "OK")
+            @ApiResponse(code = 200, message = "OK")
```

Comment:
What code style are you using? Please make sure to use that one https://github.com/TAI-EPAM/TAI-Code-Style/blob/master/intellij-java-tai-style.xml